acquire/acquire.py

```diff
@@ -1449,10 +1449,24 @@ def private_key_filter(path: fsutil.TargetPath) -> bool:
 @register_module("--home")
 class Home(Module):
     SPEC = [
+        # Catches most shell related configuration files
         ("glob", ".*[akz]sh*", from_user_home),
-        ("dir", ".config", from_user_home),
         ("glob", "*/.*[akz]sh*", from_user_home),
+        # Added to catch any shell related configuration file not caught with the above glob
+        ("glob", ".*history", from_user_home),
+        ("glob", "*/.*history", from_user_home),
+        ("glob", ".*rc", from_user_home),
+        ("glob", ".*_logout", from_user_home),
+        # Miscellaneous configuration files
+        ("dir", ".config", from_user_home),
         ("glob", "*/.config", from_user_home),
+        ("glob", "*/.wget-hsts", from_user_home),
+        ("file", ".gitconfig", from_user_home),
+        ("glob", "*/.gitconfig", from_user_home),
+        ("file", ".selected_editor", from_user_home),
+        ("file", ".viminfo", from_user_home),
+        ("file", ".lesshist", from_user_home),
+        ("file", ".profile", from_user_home),
```

Comment:
```suggestion
        ("file", ".selected_editor", from_user_home),
        ("file", "*/.selected_editor", from_user_home),
        ("file", ".viminfo", from_user_home),
        ("file", "*/.viminfo", from_user_home),
        ("file", ".lesshist", from_user_home),
        ("file", "*/.lesshist", from_user_home),
        ("file", ".profile", from_user_home),
        ("file", "*/.profile", from_user_home),
```

Reply:
Fixed the suggestion types. I don't have a strong preference for either. Adding code to `get_spec_additions` feels like you might as well try to fix the actual underlying issue. This is hackier but works around the issue minimally.